### ISSUE TITLE
[Question] What audio files are used in your mod when you have mid and high insanity?

### ISSUE THREAD

**Issue by SrMoroxide** [BudgetAirpods/InsanityRemastered#13]
Hello!
I am the creator of the AbyssalAudio mod, and I have received several recommendations to create compatibility with the audios of your mod (also to correct a small detail that the CustomSounds mod is causing).

I would like to know if the audios used are vanilla and/or custom and what the names are so I can replace them with those from my mod.

Thank you very much in advance for reading this message

**Comment by BudgetAirpods** [comment]
All of the sounds are custom made(excluding the vanilla sfx of course) and are loaded in with assets bundles and saved in a list inside the Plugin script.

Most of the sfx you hear go by a variation of "HallucinationBunker_ #".  I can send a screenshot of all of the file names so you can get the precise names down,
Though an issue I see with this is that this will need to be done every time I add more sounds.

Just curious, is my mod breaking yours or anything?

**Comment by SrMoroxide** [comment]
No, the opposite, the bug is caused by a problem in the latest version of Lethal Company.
So I'm looking for ways to change it to make it happen and your md recommended me.

**Comment by BudgetAirpods** [comment]
Sorry for the late response. I just woke up. I'm assuming you wanted to replace the SFX of IR with your own. In which case I'm not an expert at but I would assume you overwrite any of the AudioClip arrays in the Plugin class(which I'm realizing isn't in the GitHub) through an assetbundle.  A way you could get access to that array is shown in Mod Integration folder.

If I'm wrong in my assumption, please do let me know. Like I said, I just woke up and might be a little slow.
